README.rst

```diff
@@ -1,95 +1,167 @@
-=========
+#########
 I Promise
-=========
+#########
+
 .. image:: https://badge.fury.io/py/ipromise.svg
     :target: https://badge.fury.io/py/ipromise
 
-This repository provides a Python base class, and various decorators for specifying promises relating to inheritance.
-It provides three inheritance patterns:
+This repository, ``ipromise``, provides a Python base class and
+decorators to guarantee inheritance "promises".
```

Comment:
Okay, this is good, but no need for the quotations around "promises".

Reply:
Fixed.